src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/V2/DownloadedPackagesSQLLiteStoreV2.swift

```diff
@@ -546,8 +550,8 @@ private extension FMResultSet {
 extension DownloadedPackagesSQLLiteStoreV2 {
 	convenience init(fileName: String) {
 
-		let fileManager = FileManager()
-		guard let documentDir = fileManager.urls(for: .applicationSupportDirectory, in: .userDomainMask).first else {
+		let fileManager = FileManager.default
+		guard let documentDir = try? fileManager.url(for: .applicationSupportDirectory, in: .userDomainMask, appropriateFor: nil, create: true) else {
```

Comment:
does this fail silently when using `try? `but no declare the func `throws`? Or am I mistaken here?

Reply:
`fatalError` isn't silent from my pov :)

Reply:
No it does not because the guard statement handles this